infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -78,6 +78,9 @@ type FeatureStoreServices struct {
 	DeploymentStrategy *appsv1.DeploymentStrategy `json:"deploymentStrategy,omitempty"`
 	// Disable the 'feast repo initialization' initContainer
 	DisableInitContainers bool `json:"disableInitContainers,omitempty"`
+	// Volumes specifies the volumes to mount in the FeatureStore deployment. A corresponding `VolumeMount` should be added to whichever feast service(s) require access to said volume(s).
+	//+optional
```

Comment:
unnecessary, `omitempty` in the below field opt indicates this is optional, so it's treated as such
```suggestion
```